tests/0039-event.c

```diff
@@ -190,7 +191,10 @@ int main_0039_event_log(int argc, char **argv) {
         /* Create kafka instance */
         rk     = test_create_handle(RD_KAFKA_PRODUCER, conf);
         eventq = rd_kafka_queue_get_main(rk);
-        rd_kafka_set_log_queue(rk, eventq);
+        err    = rd_kafka_set_log_queue(rk, eventq);
+
+        if (err != RD_KAFKA_RESP_ERR_NO_ERROR)
```

Comment:
nit: `TEST_ASSERT(!err, "set_log_queue failed: %s", rd_kafka_err2str(err));`
or even simpler:  `TEST_CALL_ERR__(rd_kafka_set_log_queue(rk, eventq));`

Reply:
Thanks, changed it accordingly